packages/css/src/dialog/dialog.scss

```diff
@@ -0,0 +1,107 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../utils/breakpoint";
+
+.amsterdam-dialog {
+  align-content: start;
+  background-color: var(--amsterdam-dialog-background-color);
+  border: var(--amsterdam-dialog-border);
+  display: grid;
+  inset: 0;
+  max-inline-size: var(--amsterdam-dialog-max-inline-size);
+  overflow: hidden;
+  padding-block: 0;
+  padding-inline: 0;
+  position: fixed;
+
+  /* no token because dialog does not inherit from any element */
+  &::backdrop {
+    background: #0006;
+  }
+
+  &:not([open]) {
+    opacity: 0%;
+    pointer-events: none;
+  }
+}
+
+.amsterdam-dialog__form {
+  align-items: start;
+  display: grid;
+  gap: var(--amsterdam-dialog-form-gap);
+  grid-template-rows: auto 1fr auto;
+  max-block-size: var(--amsterdam-dialog-form-max-block-size);
+  padding-block: var(--amsterdam-dialog-form-padding-block);
+  padding-inline: var(--amsterdam-dialog-form-padding-inline);
+}
+
+.amsterdam-dialog__article {
+  display: grid;
+  gap: 1.5rem;
+  justify-items: flex-start;
+  max-block-size: 100%; /* safari */
+  overflow-y: auto;
+  overscroll-behavior-y: contain;
+  padding-inline-end: var(--amsterdam-dialog-article-padding-inline-end);
+
+  &::-webkit-scrollbar-track {
+    background: var(--amsterdam-dialog-background-color);
+  }
```

Comment:
What does this do? I generally wouldn't try to style scrollbars, it can have a11y implications. Also, this doesn't work on Chrome / Windows